app.ts

```diff
@@ -0,0 +1,187 @@
+"use strict";
+window.onload = () => { onloadValues() };
+window.onresize = () => { fromAndToValuesAdjustment() };
+
+const rowHeight = 49 + 15;
+let from = 0;
+let to = 0;
+let fittableRows = 0;
+
+/**
+ * calculates the amount of rows that can fit on the screen given its current height
+*/
+function amountOfRowsThatFit() {
+
+	let buttonEl = (<HTMLInputElement>document.getElementById("previousPage"))
+
+	let windowHeigt = window.innerHeight;
+	let buttonDivheigth = 57;
+	let adaptedHeight = windowHeigt - buttonDivheigth;
+
+	return Math.floor((adaptedHeight / rowHeight) - 1);
+
+}
+
+/**
+*These are the starting values for when the screen loads
+*/
+function onloadValues() {
+	let toValue = (<HTMLInputElement>document.getElementById("toValue")).value;
+	let fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value;
+
+	from = 0;
+	fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value = from.toString();
+	let rowsFittable = amountOfRowsThatFit();
+	to = from + rowsFittable;
+	toValue = (<HTMLInputElement>document.getElementById("toValue")).value = to.toString();
+	callHeaders(from, to);
+}
+
+/**
+ *Adjust the values in the start and end boxes according to the screen heigth whenever it resizes
+ */
+function fromAndToValuesAdjustment() {
+	let toValue = (<HTMLInputElement>document.getElementById("toValue")).value;
+	let fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value;
+
+	from = parseInt(fromValue);
+	let rowsThatFit = amountOfRowsThatFit();
+	to = from + rowsThatFit;
+
+	if (to > 100000) {
+		window.alert("Nee, stout >.<,your search has exceeded the maximum rows")
+		from = 100000 - rowsThatFit;
+		to = 100000;
+	} else if (from < 0) {
+		window.alert("Nee, stout >.<,your search has exceeded the minimum rows")
```

Comment:
A few points here: 
- Pls shorten your error message.
- I would rather you not use alerts to get this message across, because it tends to be quite jarring to the user experience of an app
- If you are going to print the same configuration of messages (the principle of not writing the same line of code multiple times), pls extract this error message to a function.